seerep_srv/seerep_core_fb/src/core_fb_conversion.cpp

```diff
@@ -125,6 +125,164 @@ seerep_core_msgs::QueryTf CoreFbConversion::fromFb(const seerep::fb::TransformSt
   return queryTf;
 }
 
+seerep_core_msgs::camera_intrinsics CoreFbConversion::fromFb(const seerep::fb::CameraIntrinsics& ci)
+{
+  seerep_core_msgs::camera_intrinsics ciCore;
+
+  CoreFbConversion::fromFbDataHeader(ci.header(), ciCore.header, seerep_core_msgs::Datatype::Unknown);
+
+  ciCore.height = ci.height();
+  ciCore.width = ci.width();
+  ciCore.distortion_model = ci.distortion_model()->str();
+
+  // traverse distortion list and convert
+  if (flatbuffers::IsFieldPresent(&ci, seerep::fb::CameraIntrinsics::VT_DISTORTION))
+  {
+    for (auto distortion_val : *ci.distortion())
+    {
+      ciCore.distortion.push_back(distortion_val);
+    }
+  }
+
+  // traverse intrinsics matrix and convert
+  if (flatbuffers::IsFieldPresent(&ci, seerep::fb::CameraIntrinsics::VT_INTRINSIC_MATRIX))
+  {
+    for (auto intrinsic_matrix_elem : *ci.intrinsic_matrix())
+    {
+      ciCore.intrinsic_matrix.push_back(intrinsic_matrix_elem);
+    }
+  }
+
+  // traverse rectification matrix and convert
+  if (flatbuffers::IsFieldPresent(&ci, seerep::fb::CameraIntrinsics::VT_RECTIFICATION_MATRIX))
+  {
+    for (auto rectification_matrix_elem : *ci.rectification_matrix())
+    {
+      ciCore.rectification_matrix.push_back(rectification_matrix_elem);
+    }
+  }
+
+  // traverse projection matrix and convert
+  if (flatbuffers::IsFieldPresent(&ci, seerep::fb::CameraIntrinsics::VT_PROJECTION_MATRIX))
+  {
+    for (auto projection_matrix_elem : *ci.projection_matrix())
+    {
+      ciCore.projection_matrix.push_back(projection_matrix_elem);
+    }
+  }
+
+  ciCore.binning_x = ci.binning_x();
+  ciCore.binning_y = ci.binning_y();
```

Comment:
The value of these two also do not fit the values defined in gRPC_fb_addCameraIntrinsics.py. This may be related to the issue with the matrices.